### PR TITLE
[Unit test] Fix sonic config engine unit test failure

### PR DESCRIPTION
**Pull request by stephenxs**
#### Why I did it

Fix sonic-config-engine unit test failure

Signed-off-by: Stephen Sun <stephens@nvidia.com>

#### How I did it

- Do not use pytest fixture in the test since it is not compatible with `unittest` framework which is used by all of the rest test cases.
- Supply 2 missing files

#### How to verify it

Run unit test or compile the module (when the unit test will run automatically)

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [x] 202111

### CODE REVIEW

File: src/sonic-config-engine/tests/test_j2files.py [hunk 1]
```diff
@@ -249,6 +242,18 @@ def test_buffers_render_template(self, sku_information):
...
+    def test_buffers_dell6100_render_template(self):
+        self._test_buffers_render_template('dell', 'x86_64-dell_s6100_c2538-r0', 'Force10-S6100', 'sample-dell-6100-t0-minigraph.xml', 'buffers.json.j2', 'buffers-dell6100.json')
+
+    def test_buffers_mellanox2700_render_template(self):
+        self._test_buffers_render_template('mellanox', 'x86_64-mlnx_msn2700-r0', 'Mellanox-SN2700-D48C8', 'sample-mellanox-2700-t0-minigraph.xml', 'buffers.json.j2', 'buffers-mellanox2700.json')
+
+    def test_buffers_mellanox2700_render_template(self):
+        self._test_buffers_render_template('mellanox', 'x86_64-mlnx_msn2410-r0', 'ACS-MSN2410', 'sample-mellanox-2410-t1-minigraph.xml', 'buffers.json.j2', 'buffers-mellanox2410.json')
+
+    def test_buffers_mellanox2700_render_template(self):
```
Comment (neethajohn) [inline src/sonic-config-engine/tests/test_j2files.py:254]: can you change the function name?

Comment (stephenxs) [reply src/sonic-config-engine/tests/test_j2files.py]: Fixed.

**Review by neethajohn — APPROVED** [review]

**Comment by stephenxs** [comment]
Failed by network issue. Retried.
```
2021-12-08T02:03:57.6027366Z Cloning into 'isc-dhcp'...
2021-12-08T02:03:57.6029113Z fatal: unable to access 'https://salsa.debian.org/dhcp-team/isc-dhcp.git/': Failed to connect to salsa.debian.org port 443: Connection timed out
2021-12-08T02:03:57.6030751Z make[1]: *** [Makefile:9: /sonic/target/debs/buster/isc-dhcp-relay_4.4.1-2_amd64.deb] Error 128
```

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**